portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,11 +34,9 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private List<Map<String,String>> comments = new ArrayList<Map<String,String>>();
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    comments.clear();
+    private List<Map<String,String>> comments = new ArrayList<Map<String,String>>();
```

Comment:
Do we need the "private" specifier since this is inside the function? I think this would result in a compilation error.

Reply:
I changed it!